tools/scripts/lib_install/fmtlib.sh

```diff
@@ -64,7 +64,7 @@ fi
 cd ${extracted_dir}
 mkdir -p cmake-build-release
 cd cmake-build-release
-cmake -DCMAKE_POSITION_INDEPENDENT_CODE=ON ../
+cmake ../
```

Comment:
PIC needed for `catch2` dependency.
```suggestion
cmake -DCMAKE_POSITION_INDEPENDENT_CODE=ON ../
```